model/src/plugin_handler.cpp

```diff
@@ -773,16 +773,31 @@ bool PluginHandler::archive_check(int r, const char* msg, struct archive* a) {
   return r >= ARCHIVE_WARN;
 }
 
+/*
+ * Helper function to extract a tarball into platform-specific user directories.
```

Comment:
This comment lacks a leading /**, doxygen will thus ignore it

Reply:
fixed